example/components/SoundOutputDevicesSection.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {Text, TouchableOpacity, FlatList, StyleSheet, View} from 'react-native';
+import {
+  AudioOutputDevice,
+  AudioOutputDeviceType,
+} from '@jellyfish-dev/react-native-client-sdk';
+import {TextColors} from '../utils/Colors';
```

Comment:
maybe not in this PR but in the future how about using aliases like this:
`import { TextColors } from '@colors'`

Reply:
Also do we have a ts formatter set up?

Reply:
in package.json we have specified eslingConfig

Reply:
but is it working? no spaces around TextColor looks weird

Reply:
It is working and this always has been working like this